test/test_public_bindings.py

```diff
@@ -307,41 +307,74 @@ def test_module(modname):
                     return
 
             # verifies that each public API has the correct module name and naming semantics
-            def check_one_element(elem, modname, mod, *, is_public):
+            def check_one_element(elem, modname, mod, *, is_public, is_all):
                 obj = getattr(mod, elem)
                 if not (isinstance(obj, Callable) or inspect.isclass(obj)):
                     return
                 elem_module = getattr(obj, '__module__', None)
+                # Only used for nice error message below
+                why_not_looks_public = ""
+                if elem_module is None:
+                    why_not_looks_public = "because it does not have a `__module__` attribute"
```

Comment:
this is not so helpful because a missing `__module__` attribute is typically some sort of C binding stuff is going on

Reply:
We fixed these for _C and _VF in torch. and pybind is setting that up properly. So I don't expect this error to be raised.